gtest_parallel_tests.py

```diff
@@ -109,7 +109,7 @@ def execute_tasks(self, tasks, retries, expected_exit_code):
     for test_id, expected in tasks:
       task = task_mock_factory.get_task(test_id)
       task_manager.run_task(task)
-      expected['execution_number'] = range(len(expected['exit_code']))
+      expected['execution_number'] = list(range(len(expected['exit_code'])))
```

Comment:
last nag I think, is this list conversion necessary?

Reply:
Yes, the tests use `assertListEqual` and would otherwise fail.